GameBoard/Models/FriendsSidebar/FriendsSidebarViewModel.cs

```diff
@@ -1,27 +1,43 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
-using GameBoard.Models.User;
+using GameBoard.Models.Groups;
 
 namespace GameBoard.Models.FriendsSidebar
 {
     public class FriendsSidebarViewModel
     {
-        public IEnumerable<UserViewModel> Friends { get; }
+        public IEnumerable<GroupViewModel> Groups { get; }
 
         public bool Toggled { get; }
 
+        public bool GroupInviteEnabled { get; }
+
+        public int GameEventId { get; }
+
+        public string GameEventName { get; }
+
+        public int GroupId { get; }
+
         public string SubComponentName { get; }
 
-        public Func<string, object> SubComponentArguments { get; }
+        public Func<string, int, object> SubComponentArguments { get; }
```

Comment:
We should have a delegate for this, to name the parameters. It's not clear at first glance what are the `string, int` parameters supposed to represent.

Reply:
Reverted